python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -232,20 +232,20 @@ def post(self, mine_no_or_guid=None):
             self.raise_error(400, 'Error: Unexpected mine number in Url.'), 400
 
         data = self.parser.parse_args()
-        lat = data['latitude']
-        lon = data['longitude']
-        note = data['note']
+        lat = data.get('latitude')
+        lon = data.get('longitude')
+        note = data.get('note')
         location = None
         mine_region = None
-        status = data['mine_status']
-        major_mine_ind = data['major_mine_ind']
-        mine_region = data['mine_region']
+        status = data.get('mine_status')
+        major_mine_ind = data.get('major_mine_ind')
+        mine_region = data.get('mine_region')
         mine = Mine(mine_guid=uuid.uuid4(), **self.get_create_update_dict())
```

Comment:
Why do we create a mine here?

Reply:
It's a `POST` request to the mine endpoint. That's the expected behaviour, no?

Reply:
🤷‍♂️ I didn't look into why the logic is happening 😅 but you can make a note or ticket and we can address that